app/routes.py

```diff
@@ -1,2 +1,234 @@
-from flask import Blueprint
+from app import db
+import requests
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify
+from sqlalchemy import asc, desc
+from datetime import datetime
+import os
+from dotenv import load_dotenv
+from dateutil.parser import parse
 
+load_dotenv()
+
+#################### Routes for Tasks ####################
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["GET"], strict_slashes=False)
+def get_tasks():
```

Comment:
I suggest adding some blank lines between logical sections of this function, just for readability.